modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm

```diff
@@ -187,27 +194,27 @@
 		// The gladiator hates non-humans, he especially hates ash walkers.
 		if(targetspecies.id == SPECIES_HUMAN)
 			var/static/list/human_messages = list(
-									"When you are ready, you may end the madness. Give me an honorable death.", \
-									"Only you can send me to the necropolis with honor. Will you?", \
-									"I have done all that can be done. There is nothing left. You are my last challenge.", \
-									"Beware that, when fighting monsters, you yourself do not become a monster... \
-									For when you gaze long into the abyss. The abyss gazes also into you.")
+									"Is this all that is left?", \
+									"Show the necropolis it was wrong to choose me.", \
+									"Ironic that I become what I once fought like you.", \
+									"Sometimes, the abyss gazes back.", \
+									"Show me a good time, miner!")
 			say(message = pick(human_messages))
 			introduced |= WEAKREF(target)
 		else if(targetspecies.id == SPECIES_LIZARD_ASH)
 			var/static/list/ashie_messages = list(
-									"Another slow wit comes to die!",\
-									"Let my blade help you to see, walker!",\
-									"Go back to your camp, farm tool!", \
-									"Die, fetcher!")
+									"Foolishness, ash walker!",\
+									"I've had enough of you for a lifetime!",\
+									"I don't need a crusher to KICK YOUR ASS!", \
+									"GET OVER HERE!!")
 			say(message = pick(ashie_messages), language = /datum/language/draconic)
 			introduced |= WEAKREF(target)
 			get_angry()
 			GiveTarget(target)
 		else
 			var/static/list/other_humanoid_messages = list(
 									"I will smite you!", \
-									"I will make you squeal!", \
+									"I will show you true power!", \
 									"Let us see how worthy you are!", \
 									"You will make a fine rug!", \
 									"For the necropolis!")
```

Comment:
Same here, as the other last two comments, can't make a suggestion for it sadly.